save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/inputform/InputForms.kt

```diff
@@ -161,6 +161,8 @@ enum class InputTypes(
         placeholder = SEVERITY_VECTOR_PLACEHOLDER,
         tooltip = SEVERITY_VECTOR_TOOLTIP,
     ),
+    VULN_CREDIT_NAME("user name", placeholder = "User name"),
+    VULN_CREDIT_CONTACTS("contacts", placeholder = "Contacts"),
```

Comment:
I think it’s better to explicitly list in the placeholder what the user can enter. `Url, mail, social networks, etc.`

Reply:
actually yes, OSV supports different type of credits:
```json
    "type": {
            "type": "string",
            "enum": [
              "FINDER",
              "REPORTER",
              "ANALYST",
              "COORDINATOR",
              "REMEDIATION_DEVELOPER",
              "REMEDIATION_REVIEWER",
              "REMEDIATION_VERIFIER",
              "TOOL",
              "SPONSOR",
              "OTHER"
          ]
```
Now we support only `null`?